plugin/cds-hooks/src/main/java/org/opencds/cqf/ruler/cdshooks/dstu3/Dstu3CqlExecution.java

```diff
@@ -0,0 +1,81 @@
+package org.opencds.cqf.ruler.cdshooks.dstu3;
+
+import ca.uhn.fhir.jpa.partition.SystemRequestDetails;
+import ca.uhn.fhir.rest.api.server.RequestDetails;
+import org.hl7.fhir.dstu3.model.BooleanType;
+import org.hl7.fhir.dstu3.model.Bundle;
+import org.hl7.fhir.dstu3.model.CodeableConcept;
+import org.hl7.fhir.dstu3.model.Endpoint;
+import org.hl7.fhir.dstu3.model.IdType;
+import org.hl7.fhir.dstu3.model.OperationOutcome;
+import org.hl7.fhir.dstu3.model.Parameters;
+import org.hl7.fhir.dstu3.model.Type;
+import org.opencds.cqf.ruler.cpg.dstu3.provider.CqlExecutionProvider;
+import org.opencds.cqf.ruler.cpg.dstu3.provider.LibraryEvaluationProvider;
+
+import java.util.List;
+import java.util.Optional;
+
+public class Dstu3CqlExecution {
```

Comment:
There's an CqlExceptionHandlingInterceptor that has some of this logic, I think.